hot/log.js

```diff
@@ -0,0 +1,14 @@
+var logLevel = "info";
+
+module.exports = function(level, msg) {
+	if(logLevel === "info" && level === "info")
```

Comment:
I just copy/pasted the log() function from webpack-dev-server/client/index.js.
The switch doesn't work in all cases, because the log levels are inclusive - logLevel === "warning" && level === "info" should console.log(), but the switch does nothing.